application/Common/Shared/Utility/NativeMethods.cs

```diff
@@ -76,6 +76,38 @@ public static bool IsKeyPressed(VirtualKeyCode virtualKeyCode)
 
         #endregion
 
+        #region Clipboard text helper
+
+        /// <summary>
+        ///     Gets the text from the clipboard
+        /// </summary>
+        /// <param name="hwnd">Pointer to the window that currently has clipboard</param>
+        /// <returns>String representing text from the clipboard</returns>
+        public static string getClipboardText()
+        {
+            uint CF_TEXT = 1;
+            uint CF_UNICODETEXT = 13;
```

Comment:
This could be converted to a constant. I would instead suggest moving this into an enum with the other possible clipboard formats.